migrations/3_20190212095852_cards.js

```diff
@@ -16,8 +16,8 @@ exports.up = function(knex, Promise) {
     table
       .integer('created_by')
       .references('id')
-      .inTable('users')
-      .notNullable();
+      .inTable('users');
+    // .notNullable();
```

Comment:
Any reason why this is nullable?

Reply:
leaving as nullable for now, for easier route testing before integrating users